docs/modules/ROOT/pages/getting-started/your-first-query.adoc

```diff
@@ -0,0 +1,80 @@
+= Your first query
+
+In this guide, you'll create a simple Cypher query to match and return Movies using the Cypher Builder:
+
+```cypher
+MATCH(m:Movie)
+RETURN m
+```
+
+== Matching nodes
```

Comment:
```suggestion
```

Reply:
Isn't it a good idea to split the guide into separate small sections?